### PR TITLE
Add template for documentation issues

### PR DESCRIPTION
**Pull request by rachel-fenichel**
Add a template asking the right questions for documentation issues.

I will also add this to master.

### CODE REVIEW

**Comment by rachel-fenichel** [comment]
Added a line saying to link to a PR.

**Comment by rachel-fenichel** [comment]
@RoboErikG for any comments before I merge this.

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - Use this template to describe issues with the content at
+  - developers.google.com/blockly/guides
+  -->
+
+**Where**
+
+<!-- A link to the page with the documentation you want us to update.
+  -  More specific is better.  If no page exists, describe what the page
+  -  should be, and where.
+  -->
```
Comment (RoboErikG) [inline .github/ISSUE_TEMPLATE/documentation.md]: It might be nice to add some checkboxes for 'What'
[ ] Text
[ ] Image or Gif
[ ] Other

Comment (rachel-fenichel) [reply .github/ISSUE_TEMPLATE/documentation.md]: Seems redundant, since I'm asking them to put that text or gif in the issue as well.

Comment (RoboErikG) [reply .github/ISSUE_TEMPLATE/documentation.md]: As is I read it only as asking for the text. I don't see anything that specifically asks them to note if we need to update an image.

Comment (rachel-fenichel) [reply .github/ISSUE_TEMPLATE/documentation.md]: Okay, updating and I'll add it to my cherry-pick as well.

**Pull request merged**